### PR TITLE
Refactor singularity help pull options text.

### LINKED ISSUE #2986: refactory `singularity help pull` options text
## Version of Singularity:

3.1.0

* `interactive prompt for docker authentication`

change to: `Login to a Docker Repository interactively`

*
<pre>
the library to pull from (default
                         "https://library.sylabs.io")
</pre>

change to:

`Download images from the provided library (default is "https://library.sylabs.io")`

Try to put that all on one line.

* 
<pre>
do NOT use HTTPS, for communicating with local
                         docker registry
</pre>

change to:

`Do not use HTTPS with the docker:// transport (useful for local docker registries without a certificate)`

Try to put that all on one line

### PR DESCRIPTION
**Pull request by gvallee**
**Description of the Pull Request (PR):**

Refactor singularity help pull options text.
Fixes #2986. 

**This fixes or addresses the following GitHub issues:**

- Fixes #2986 

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: cmd/internal/cli/action_flags.go [hunk 1]
```diff
@@ -229,7 +229,7 @@ func initBoolVars() {
 	actionFlags.SetAnnotation("nohttps", "envkey", []string{"NOHTTPS"})
 
 	// --docker-login
-	actionFlags.BoolVar(&dockerLogin, "docker-login", false, "interactive prompt for docker authentication")
+	actionFlags.BoolVar(&dockerLogin, "docker-login", false, "Login to a Docker Repository interactively")
```
Comment (sashayakovtseva) [inline cmd/internal/cli/action_flags.go]: Shouldn't it start with a lowercase letter for consistency as everywhere else?

**Review by sashayakovtseva — CHANGES_REQUESTED** [review]

File: cmd/internal/cli/action_flags.go (hunk 1 above)
Comment (gvallee) [reply cmd/internal/cli/action_flags.go]: Fixed

File: cmd/internal/cli/pull.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ var (
 func init() {
 	PullCmd.Flags().SetInterspersed(false)
 
-	PullCmd.Flags().StringVar(&PullLibraryURI, "library", "https://library.sylabs.io", "the library to pull from")
+	PullCmd.Flags().StringVar(&PullLibraryURI, "library", "https://library.sylabs.io", "Download images from the provided library")
```
Comment (sashayakovtseva) [inline cmd/internal/cli/pull.go]: Same here, lowecase

File: cmd/internal/cli/pull.go [hunk 3]
```diff
@@ -60,7 +60,7 @@ func init() {
 	PullCmd.Flags().Lookup("tmpdir").Hidden = true
 	PullCmd.Flags().SetAnnotation("tmpdir", "envkey", []string{"TMPDIR"})
 
-	PullCmd.Flags().BoolVar(&noHTTPS, "nohttps", false, "do NOT use HTTPS, for communicating with local docker registry")
+	PullCmd.Flags().BoolVar(&noHTTPS, "nohttps", false, "Do NOT use HTTPS with the docker:// transport (useful for local docker registries without a certificate)")
```
Comment (sashayakovtseva) [inline cmd/internal/cli/pull.go]: Also lowercase

**Review by sashayakovtseva — CHANGES_REQUESTED** [review]
Sorry, I didn't request changes everywhere at the first time, thought it was implied :)

File: cmd/internal/cli/pull.go (hunk 2 above)
Comment (gvallee) [reply cmd/internal/cli/pull.go]: fixed

File: cmd/internal/cli/pull.go (hunk 3 above)
Comment (gvallee) [reply cmd/internal/cli/pull.go]: fixed

**Comment by gvallee** [comment]
I am the one who is sorry, i should have caught that. Thanks for the feedback!

**Review by sashayakovtseva — APPROVED** [review]

**Pull request merged**